src/services/trip.service.js

```diff
@@ -31,6 +31,22 @@ class TripService {
       where: { ...property }
     });
   }
+
+  /**
+ *
+ * @static
+ * @param {property} userId
+ * @memberof TripService
+ * @returns {object} data
+ */
+  static userRequestLists(userId) {
+    return Trip.findAll({
```

Comment:
Can you make this function dynamic? so that it can be reusable for future search queries in trips folder. Please refer to the `user.service.js` file.